bin/python-noenv.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-env -i /usr/bin/python "$@"
+env -i PBS_DEFAULT=$PBS_DEFAULT /usr/bin/python "$@"
```

Comment:
the list of vars that should be set is also not that trivial; and don't forget we should still expect things like `os.system` to work